packages/react-native-editor/ios/GutenbergDemo.xcodeproj/project.pbxproj

```diff
@@ -764,7 +764,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
```

Comment:
These changes don't affect the release since we are constantly changing them 😅 so it's fine to leave it as it is. What do you think?

Reply:
Yeah, this was probably caused by using different CPU architectures when running the release script (i.e. ARM vs Intel) 😞. I'm still on Intel so that would explain why it was changed.